git_machete/client.py

```diff
@@ -1842,7 +1842,8 @@ def checkout_github_prs(self,
                             all_opened_prs: bool = False,
                             my_opened_prs: bool = False,
                             opened_by: str = None,
-                            verbose: bool = False
+                            verbose: bool = False,
+                            called_function: bool = True
```

Comment:
Let's name the param more explicitly (so that the param name already tells what's the effect &mdash; now one would need to check the implementation):
```suggestion
                            fail_on_missing_current_user_for_my_opened_prs: bool = True
```
Verbose but unambiguous